ui/admin/tests/acceptance/groups/create-test.js

```diff
@@ -52,11 +49,14 @@ module('Acceptance | groups | create', function (hooks) {
   });
 
   test('can create new group', async function (assert) {
-    const groupsCount = this.server.db.groups.length;
+    const groupsCount = this.server.schema.groups.all().models.length;
     await visit(urls.newGroup);
-    await fillIn('[name="name"]', 'group name');
-    await click('[type="submit"]');
-    assert.strictEqual(this.server.db.groups.length, groupsCount + 1);
+    await fillIn(commonSelectors.FIELD_NAME, commonSelectors.FIELD_NAME_VALUE);
```

Comment:
None blocking comment: since a group has also the field Description as optional, like name, I will suggest to also `fillIn` the field description.